MultiplayerMod/Multiplayer/Objects/MultiplayerIdentityProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MultiplayerMod.Game.World;
+
+namespace MultiplayerMod.Multiplayer.Objects;
+
+public static class MultiplayerIdentityProvider {
+
+    private static long nextId;
+
+    public static long GetNextId() => nextId++;
+
+    static MultiplayerIdentityProvider() {
```

Comment:
Usually I'm getting quite wary seeing static constructor especially with event subscription.
Is it possible to invert this in different way? 
E.g. subscribe in MultiplayerObjectsConfigurator and call MultiplayerIdentityProvider.SetInitialId ?

Reply:
Yep, it's just a template, because, well, it's a draft.
(Btw yeah, static constructors in C# can lead to weird results).

Reply:
Yep, as I expected the real implementation turned into a singleton, so no worries about that.